packages/atomic/src/pages/index.html

```diff
@@ -356,7 +356,7 @@
                     </atomic-result-section-excerpt>
                   </atomic-table-element>
                   <atomic-table-element label="author">
-                    <atomic-result-text field="author"></atomic-result-text>
+                    <atomic-result-multi-value-text field="author"></atomic-result-multi-value-text>
```

Comment:
for some reason, when selecting some automatic facets, it logged an error in the console because of multi value being used in a normal result-text.

Reply:
would be interesting to know why :P

Reply:
Last friday, I was playing with the platform api to modify fields in the index and I kept using the `author` field for my examples. I must have changed a parameter in the `author` field. 🤔 🤔 Jlessard was also making changes to the searchuisamples org with the api. Anyway, this problem is not caused by the automatic facets. Its something about a change in the certain field.